### PR TITLE
Allow rewrite of Observer class

### PR DESCRIPTION
**Pull request by shineability**
These changes allow you to rewrite the Observer class so the releaseStage() can be implemented in a custom way...

### CODE REVIEW

**Comment by shineability** [comment]
Anyone? :)

**Comment by shineability** [comment]
Anybody had a look at this yet? :)

File: src/app/code/community/Bugsnag/Notifier/etc/config.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <observers>
           <bugsnag_controller_action_predispatch>
             <type>singleton</type>
-            <class>Bugsnag_Notifier_Model_Observer</class>
+            <class>Bugsnag_Notifier/Observer</class>
```
Comment (GrahamCampbell) [inline src/app/code/community/Bugsnag/Notifier/etc/config.xml:19]: What does this do?

Comment (shineability) [reply src/app/code/community/Bugsnag/Notifier/etc/config.xml:19]: @GrahamCampbell This way you can rewrite the observer class if needed which is not possible if you don't use a grouped class name (if i'm not mistaking). E.g. if you want to define custom logic to define what environment you're using...

```
protected function releaseStage()
{
    return $_ENV['environment'];
}
```

Comment (shineability) [reply src/app/code/community/Bugsnag/Notifier/etc/config.xml:19]: @GrahamCampbell Wdyt? Would be nice to see this merged, we use this package in all of our Magento shops...

**Comment by luke-belton** [comment]
`bugsnag-magento` has been deprecated.

**Pull request closed without merging**